server/src/internalClusterTest/java/org/elasticsearch/search/fetch/subphase/InnerHitsIT.java

```diff
@@ -195,12 +195,12 @@ public void testRandomNested() throws Exception {
                     .field("foo", i)
                     .startArray("field1");
             for (int j = 0; j < numInnerObjects; j++) {
-                source.startObject().field("x", "y").endObject();
+                source.startObject().field("x", "y" + i + ":" + j).endObject();
```

Comment:
I used these to debug some deep weirdness with how InnerHits uses its lookups, but they're useful so I think they can stay